pkg/webhooks/sdnmigration/sdnmigration.go

```diff
@@ -18,14 +18,13 @@ import (
 const (
 	WebhookName               string = "sdn-migration-validation"
 	privilegedServiceAccounts string = `^system:serviceaccounts:(kube.*|openshift.*|default|redhat.*|osde2e-[a-z0-9]{5})`
-	privilegedUsers           string = `system:admin`
 	docString                 string = `Managed OpenShift customers may not modify the network config type because it can can degrade cluster operators and can interfere with OpenShift SRE monitoring.`
+	overrideAnnotation        string = "red-hat-internal-testing"
```

Comment:
Kinda nit-picky, but thoughts on updating the annotation to
```suggestion
	overrideAnnotation        string = "unsupported-red-hat-internal-testing"
```
to emphasize that this annotation should definitely not be used if you don't know what you're doing?